### PR TITLE
PMM-6944 API for k8s config.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1340

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -10,4 +10,4 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-6944-api-for-k8s-config"
```
Comment (askomorokhov) [inline Gopkg.toml:13]: 👀

**Review by askomorokhov — APPROVED** [review]

File: management/dbaas/kubernetes_server_test.go [hunk 2]
```diff
@@ -95,6 +95,35 @@ func TestKubernetesServer(t *testing.T) {
...
+		unregisterKubernetesClusterResponse, err := dbaasClient.Default.Kubernetes.UnregisterKubernetesCluster(
+			&kubernetes.UnregisterKubernetesClusterParams{
+				Body:    kubernetes.UnregisterKubernetesClusterBody{KubernetesClusterName: kubernetesClusterName},
+				Context: pmmapitests.Context,
+			},
+		)
+		require.NoError(t, err)
+		assert.NotNil(t, unregisterKubernetesClusterResponse)
+
+		clusters, err := dbaasClient.Default.Kubernetes.ListKubernetesClusters(nil)
```
Comment (BupycHuk) [inline management/dbaas/kubernetes_server_test.go:122]: let's remove List from here.

File: management/dbaas/kubernetes_server_test.go [hunk 3]
```diff
@@ -95,6 +95,35 @@ func TestKubernetesServer(t *testing.T) {
...
+				KubernetesClusterName: kubernetesClusterName,
+			},
+			Context: pmmapitests.Context,
+		})
+		assert.NoError(t, err)
+		assert.NotNil(t, cluster)
+		assert.NotNil(t, cluster.Payload.KubeAuth)
+		assert.Equal(t, kubeConfig, cluster.Payload.KubeAuth.Kubeconfig)
+
+		unregisterKubernetesClusterResponse, err := dbaasClient.Default.Kubernetes.UnregisterKubernetesCluster(
```
Comment (BupycHuk) [inline management/dbaas/kubernetes_server_test.go:113]: it's also not required

**Review by BupycHuk — CHANGES_REQUESTED** [review]
unregister and list not required in Get test

File: management/dbaas/kubernetes_server_test.go (hunk 3 above)
Comment (JiriCtvrtka) [reply management/dbaas/kubernetes_server_test.go]: Removed.

File: management/dbaas/kubernetes_server_test.go (hunk 2 above)
Comment (JiriCtvrtka) [reply management/dbaas/kubernetes_server_test.go]: Removed.

**Review by BupycHuk — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**